x-pack/platform/plugins/shared/triggers_actions_ui/public/application/lib/rule_api/common_transformations.ts

```diff
@@ -88,6 +88,7 @@ export const transformRule: RewriteRequestCase<Rule> = ({
   next_run: nextRun,
   alert_delay: alertDelay,
   flapping,
+  dashboards,
```

Comment:
Not sure what exactly the task_runner does. I am wondering if actually adding dashboards here is needed?